src/parse-result.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Parse result-specific refract elements.
+ *
+ * General structure:
+ *
+ * + ParseResult
+ *   + Annotation
+ */
+
+import apiDescription from 'minim-api-description';
+
+export function namespace(options) {
+  const minim = options.base;
+  const ArrayElement = minim.getElementClass('array');
+  const StringElement = minim.getElementClass('string');
+
+  class ParseResult extends ArrayElement {
+    constructor() {
+      super(...arguments);
+      this.element = 'parseResult';
+    }
+
+    get api() {
+      return this.children(item => item.classes.contains('api')).first();
```

Comment:
Sort of, yes. Since we called `namespace.use(apiDescription)` we guarantee that those elements are loaded and that Minim will give us a `Category` element above. If we didn't do that, then the user would have to (which we decided against above).

Basically the way Minim works is that when it encounters an element it looks up what is registered in the namespace for that element and creates an instance. If nothing is found then it would just return a `BaseElement` instance. By calling `.use(..)` we guarantee which element names will have a class registered in the namespace, so we can be sure that we will get a `Category` instance.

Also, keep in mind that these can be changed later, e.g. a user could load this namespace and then register their own overrides, but so long as a class exists for the `category` element name then it will be instantiated and returned in the helper method above.